### PR TITLE
TMA-1292: K8s load and smoke spec

### PR DESCRIPTION
**Pull request by kubamahnert**

### CODE REVIEW

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-opensource-repository-check/1408/ : SUCCESS in 9s
- https://jenkins-ii.intgdc.com/job/gooddata-ruby/job/gooddata-ruby-lcm-build-pipeline/115/ : FAILURE in 11m 03s

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-opensource-repository-check/1410/ : SUCCESS in 6s
- https://jenkins-ii.intgdc.com/job/gooddata-ruby/job/gooddata-ruby-lcm-build-pipeline/116/ : FAILURE in 11m 37s

**Comment by kubamahnert** [comment]
ok to test

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-opensource-repository-check/1411/ : SUCCESS in 14s

File: spec/lcm/integration/brick_runner.rb [hunk 1]
```diff
@@ -33,13 +33,18 @@ def schedule_brick(brick_name, service_project, opts)
       decoded_params = GoodData::Helpers.decode_params(script_params(opts))
       params, hidden_params = extract_hidden_params(decoded_params)
       params.delete('gd_encoded_hidden_params')
-      appstore_name = GoodData::Environment::ConnectionHelper::LCM_ENVIRONMENT[:appstore_deploy_name]
-      deploy_string = "${#{appstore_name}}:branch/lcm2:/apps/#{brick_name}"
+      component_brick_name = brick_name.sub('_brick', '')
+      component_brick_name = 'provision' if component_brick_name == 'provisioning'
```
Comment (Hahihula) [inline spec/lcm/integration/brick_runner.rb]: we should fix this properly and not have this matching

**Comment by gdgate** [comment]
Build failed (gate pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-merge-compliance-check/305/ : FAILURE in 19s

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-merge-compliance-check/308/ : SUCCESS in 2m 57s

**Pull request merged**